src/ethereum/forks/osaka/vm/gas.py

```diff
@@ -84,6 +84,109 @@
 GAS_BLS_G2_MUL = Uint(22500)
 GAS_BLS_G2_MAP = Uint(23800)
 
+# EEST Specific Variables
+G_TX_DATA_ZERO = 4
+G_TX_DATA_NON_ZERO = 16
+
+# Transaction costs (from transactions.py)
+TX_BASE_COST = Uint(21_000)
+TX_CREATE_COST = Uint(32_000)
+TX_ACCESS_LIST_ADDRESS_COST = Uint(2_400)
+TX_ACCESS_LIST_STORAGE_KEY_COST = Uint(1_900)
+STANDARD_CALLDATA_TOKEN_COST = Uint(4)
+FLOOR_CALLDATA_COST = Uint(10)
+
+# Authorization costs (from eoa_delegation.py)
+PER_EMPTY_ACCOUNT_COST = 25_000
+PER_AUTH_BASE_COST = 12_500
+
+# Opcode Costs
+G_STOP = GAS_ZERO
+G_ADD = GAS_VERY_LOW
+G_MUL = GAS_LOW
+G_SUB = GAS_VERY_LOW
+G_DIV = GAS_LOW
+G_SDIV = GAS_LOW
+G_MOD = GAS_LOW
+G_SMOD = GAS_LOW
+G_ADDMOD = GAS_MID
+G_MULMOD = GAS_MID
+G_EXP = GAS_EXPONENTIATION
```

Comment:
Might be worth adding a docstring to each of these constants that explains the difference?